book/src/continuous_integration/github_actions.md

```diff
@@ -0,0 +1,22 @@
+# GitHub Actions
+
+## actions-rs
+
+An easy way to get started with adding Clippy to GitHub Actions is
+the [actions-rs](https://github.com/actions-rs) [clippy-check](https://github.com/actions-rs/clippy-check):
+
+```yml
+on: push
+name: Clippy check
+jobs:
+    clippy_check:
```

Comment:
NIT: Usually your indentation for those files is 2 spaces.

Reply:
Sorry, missed this one. Fixed.